src/Cake.Web/App_Data/docs/fundamentals/aliases.md

```diff
@@ -40,6 +40,17 @@ public static class MyCakeExtension
         return value? int.MinValue : int.MaxValue;
     }
 
+    [CakeMethodAlias]
+    public static int GetMagicNumberOrDefault(this ICakeContext context, bool value, Func<int> defaultValueProvider = null)
+    {
+        if (value)
+        {
+            return int.MinValue;
+        }
+
+        return defaultValueProvider == null? int.MaxValue : defaultValueProvider();        
```

Comment:
@gep13 that was intentional to match formatting of the existing example.  Based on your comment I cleaned up both examples and refreshed the commit.